frontend/src/scenes/App.tsx

```diff
@@ -135,14 +135,13 @@ function _App(): JSX.Element | null {
                     }`}
                     style={{ minHeight: '100vh' }}
                 >
-                    {!sceneConfig.hide_top_nav && featureFlags['navigation-1775'] ? <TopNavigation /> : <TopContent />}
+                    {!sceneConfig.hideTopNav && featureFlags['navigation-1775'] ? <TopNavigation /> : <TopContent />}
                     <Layout.Content className="main-app-content" data-attr="layout-content">
-                        <DemoWarning />
+                        {!sceneConfig.hideDemoWarnings && <DemoWarning />}
 
                         {!featureFlags['hide-billing-toolbar'] && <BillingToolbar />}
                         {featureFlags['navigation-1775'] ? <BackTo /> : null}
-
-                        {currentTeam && !currentTeam.ingested_event && (
+                        {currentTeam && !currentTeam.completed_snippet_onboarding && !sceneConfig.hideDemoWarnings && (
```

Comment:
I think this conditional is wrong - we only set completed_snippet_onboarding on /ingestion pages. Let's change to `team.ingested_event`?

Reply:
You're right! I think we need to have both cases. Addressing in subsequent PR.